theme/theme.inc

```diff
@@ -84,6 +84,38 @@ function template_preprocess_islandora_basic_collection_wrapper(&$variables) {
 }
 
 /**
+ * Implements hook_theme().
```

Comment:
Isn't strictly true... This is the implementation of the element themeing itself. The [implementation of `hook_theme()` is in the `.module`](https://github.com/Islandora/islandora_solution_pack_collection/pull/179/files#diff-f7990cf10d52b3d4d46d8c0e9daaafa2R200).